foundry.toml

```diff
@@ -4,9 +4,12 @@ optimizer_runs = 800
 via_ir = false
 ffi = true
 fs_permissions = [{ access = "read-write", path = ".forge-snapshots/"}, { access = "read", path = "./out"}]
-evm_version = 'paris'
+cancun = true
+solc = "./bin/solc"
 
 [profile.ci]
 fuzz_runs = 100000
+solc = "./bin/solc-static-linux"
+cancun = true
```

Comment:
the default one should override this no?

Reply:
also why is it cancun = true rather than `evm_version = cancun` which is the standard foundry config?

Reply:
I think its just not supported yet? This was a custom config they did for us 

https://github.com/foundry-rs/foundry/issues/5782